recipe/build.sh

```diff
@@ -1,5 +1,40 @@
 #!/bin/bash
-export DISABLE_AUTOBREW=1
-$R CMD INSTALL --build .
+if [[ $target_platform =~ linux.* ]] || [[ $target_platform == win-32 ]] || [[ $target_platform == win-64 ]] || [[ $target_platform == osx-64 ]]; then
+  export DISABLE_AUTOBREW=1
+  $R CMD INSTALL --build .
 
-$R -e "IRkernel::installspec(sys_prefix = TRUE)"
+  # This installs to the wrong place (/usr/local)
+  # $R -e "IRkernel::installspec(user=F)"
+
+  mkdir -p $PREFIX/share/jupyter/kernels/ir
```

Comment:
This should also be kept.

Reply:
Should we retain the mkdir and cp steps?

Reply:
Yes, exactly.